libs/template/presets/resources/icons/storefront/font-icons.ts

```diff
@@ -0,0 +1,123 @@
+    [IconTypes.DropUp]: 'expand_less', // merge with expand
+    [IconTypes.InputError]: 'error', // merge with error
+    [IconTypes.Minus]: 'remove',
+    [IconTypes.Decrease]: 'remove', // merge with minus
+    [IconTypes.Printer]: 'print',
+    [IconTypes.Trash]: 'delete',
+    [IconTypes.Rocket]: 'rocket',
+    [IconTypes.Success]: 'check_circle',
+    [IconTypes.Invisible]: 'visibility_off',
+    [IconTypes.Wishlist]: 'favorite',
+    [IconTypes.Location]: 'location_on',
+    [IconTypes.NavigationArrow]: 'arrow_right_alt',
+    [IconTypes.Actions]: 'more_vert',
+    [IconTypes.User]: 'person',
+    [IconTypes.Image]: 'photo_library',
+    [IconTypes.ModeLight]: 'light_mode',
+    [IconTypes.ModeDark]: 'dark_mode',
+    [IconTypes.Ratings]: 'reviews',
+    [IconTypes.Planet]: 'language',
+    [IconTypes.Locker]: 'lock',
+    [IconTypes.Images]: 'images',
+  },
+};
+
+export const storefrontFontAwesomeIcons: IconMapper = {
+  id: 'fa',
+  styles: {
+    font: 'Font Awesome 6 Free',
+    weight: 900,
+  },
+  mapping: {
+    [IconTypes.Parcel]: '&#xf466;',
+    [IconTypes.Visible]: '&#xf06e;',
+    [IconTypes.Invisible]: '&#xf070;',
+    [IconTypes.Profile]: '&#xf007;',
+    [IconTypes.Customer]: '&#xf007;',
+    [IconTypes.Sales]: '&#xf555;',
+    [IconTypes.Users]: '&#xf508;',
+    [IconTypes.Orders]: '&#xf218;',
+    [IconTypes.Desktop]: '&#xf390;',
+    [IconTypes.Mobile]: '&#xf3cd;',
+    [IconTypes.Tablet]: '&#xf3fa;',
+    [IconTypes.Disconnect]: '&#xe55e;',
+    [IconTypes.Video]: '&#xf03d;',
+    [IconTypes.Composition]: '&#xf00a;',
+    [IconTypes.Media]: '&#xf87c;',
+    [IconTypes.Price]: '&#xf0d6;',
+    [IconTypes.Card]: '&#xf249;',
+    [IconTypes.Top]: '&#xe4c2;',
+    [IconTypes.Bottom]: '&#xe4b8;',
+    [IconTypes.Filters]: '&#xf085;',
+    [IconTypes.BulletList]: '&#xf0ca;',
+    [IconTypes.ArrowsOutward]: '&#xf337;',
+    [IconTypes.ViewList]: '&#xf00b;',
+    [IconTypes.InputStepper]: '&#xf0d8;',
+    [IconTypes.Imports]: '&#xf019;',
+    [IconTypes.Copy]: '&#xf0c5;',
+    [IconTypes.File]: '&#xf15b;',
+    [IconTypes.Imports]: '&#xf019;',
+  },
+};
+
+export const storefrontIconSources: IconSource[] = [
+  {
+    resource: storefrontFontAwesomeIcons,
+    types: [
+      IconTypes.Parcel,
+      IconTypes.Visible,
+      IconTypes.Invisible,
+      IconTypes.Profile,
+      IconTypes.Customer,
+      IconTypes.Sales,
+      IconTypes.Users,
+      IconTypes.Orders,
+      IconTypes.Desktop,
+      IconTypes.Mobile,
+      IconTypes.Tablet,
+      IconTypes.Disconnect,
+      IconTypes.Video,
+      IconTypes.Composition,
+      IconTypes.Media,
+      IconTypes.Price,
+      IconTypes.Card,
+      IconTypes.Top,
+      IconTypes.Bottom,
+      IconTypes.Filters,
+      IconTypes.BulletList,
+      IconTypes.ArrowsOutward,
+      IconTypes.ViewList,
+      IconTypes.InputStepper,
+      IconTypes.Imports,
+      IconTypes.Copy,
+      IconTypes.File,
+      IconTypes.Imports,
+    ],
+  },
+];
+
```

Comment:
This definition should not exist, and after introducing visible and visible_off to material design I have not seen any regression. Can we remove it? Or do we depend on it in the icons for EB? If the latter is true, we can keep it for now and find a way to drop it going forward.

Reply:
In order that we use IconTypes as fallback so we should define all icons or we will see just text in storybook. I fixed it to add all definitions into materialDesign icons, as we don't use `storefront\bo` prefix know I guess it's ok when we have the full icon list of our IconTypes